pkg/anonymization/anonymizer_test.go

```diff
@@ -0,0 +1,201 @@
+package anonymization
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/insights-operator/pkg/config"
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+func TestGetNextIP(t *testing.T) {
```

Comment:
I am not sure, but @rluders suggested `Test_GetNextIP` name format or am I wrong?

Reply:
Correct.